XenModel/Actions/ZipStatusReportAction.cs

```diff
@@ -62,8 +62,14 @@ public class ZipStatusReportAction : AsyncAction
 
         private long bytesToCompress = 1;
 
-        public ZipStatusReportAction(string tempFolder, string destFile)
-            : base(null, Messages.BUGTOOL_SAVING, Messages.BUGTOOL_SAVING, true)
+        /// <summary>
+        ///  Constructor to zip downloaded file action
+        /// </summary>
+        /// <param name="tempFolder">empFolder to store the downloaded logs</param>

```

Comment:
"t" missing from tempFolder
"compressed" result
whether "to" suppress history in the "events tab"

Reply:
updated